include/lo2s/trace/trace.hpp

```diff
@@ -239,6 +243,7 @@ class Trace
                                        const otf2::definition::string& iname);
 
     void process_update_executable(pid_t pid, const otf2::definition::string& exe_name);
+    void task_update_command(pid_t tid, const otf2::definition::string& comm);
```

Comment:
I think it's nicer to group those two with the respective `const std::string&` overloads.